blueocean-core-js/src/js/NotFound.jsx

```diff
@@ -15,7 +15,7 @@ export default () => (
                 defaultValue: 'Page not found (404)',
             })}</h3>
             <div className="message">{translate('Not.found.message', { defaultValue: 'Jenkins could not find the page you were looking for. Check the URL for errors or press the back button.' })}</div>
-            <div className="actions"><Link to="/" className="btn btn-primary inverse">{translate('Open.dashboard', { defaultValue: 'Open Dashboard' })}</Link></div>
```

Comment:
will need to run ATH locally and check all the breakages (likely small changes to a few strings of what to look for). 

More seriously it just looks quite wrong when I tried it